packaging/installer-linux/installer-rpm/src/main/resources/neo4j/neo4jd

```diff
@@ -32,7 +32,7 @@ java -cp "${CLASSPATH}" \
         -Dneo4j.home="${NEO4J_HOME}" \
         -Dfile.encoding=UTF-8 \
         -Dorg.neo4j.server.properties=conf/neo4j-server.properties \
-        org.neo4j.server.Bootstrapper > /var/log/neo4j/console.log 2>&1 &
+        #{neo4j.mainClass} > /var/log/neo4j/console.log 2>&1 &
```

Comment:
No, maven has been overridden to use '#' instead, since this is for maven to replace at build time, and '${}' are used by the shell script itself